trycourier/audiences.py

```diff
@@ -0,0 +1,114 @@
+from .exceptions import CourierAPIException
+
+
+class Audiences():
+    key = "audiences"
+
+    def __init__(self, base_url, session):
+        self.base_url = base_url
+        self.session = session
+
+    @property
+    def uri(self):
+        return "%s/%s" % (self.base_url, self.key)
+
+    def put_audience(self, audience):
```

Comment:
I think here we should enforce `audienceId` as a param instead of taking it from audience object. so url construction on line 27 does not get crazy null stuff in it